src/components/Categories.js

```diff
@@ -1,9 +1,20 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { checkStatus } from '../redux/categories/categories';
 
-const Categories = () => (
-  <div className="category-container">
-    <button className="primary-button" type="button">Check status</button>
-  </div>
-);
+const Categories = () => {
+  const dispatch = useDispatch();
+  return (
+    <div className="category-container">
+      <button
+        className="primary-button"
+        type="button"
+        onClick={() => dispatch(checkStatus())}
+      >
+        Check status
+      </button>
+    </div>
```

Comment:
- [ ] Kindly consider when clicking on the check status button the message with `Under Construction` which you get from the state by using `useSelector`.

Reply:
Hi @omarsalem7 , thank you for your comment, I will add the message.